src/bitset.hpp

```diff
@@ -354,6 +411,36 @@ TEST_CASE("Bitset") {
   // "Lift" a bitset.
   CHECK_EQ(Remap(Bitset("11"), {0, std::nullopt, 1}), Bitset("101"));
 }
+
+// See simple diagram at:
+// https://user-images.githubusercontent.com/31897211/136752835-2ad4649b-dc14-4c80-8174-d0567e849cf8.gif
```

Comment:
I guess I'm not getting much out of this one. It's showing the various parent-child subsplit pairs?

Reply:
Correct.  It is just a simplified look at the input (blue) and the two different output (red) for the function.  The second diagram contextualizes them in the tree.  I can remove it if it is not helpful.  Is the full tree diagram clear?